.github/workflows/shared-publish-to-docker-versioned.yaml

```diff
@@ -72,6 +72,6 @@ jobs:
           is_release: ${{ steps.setup.outputs.is_release }}
           new_version: ${{ inputs.new_version }}
           tags: ${{ steps.publishToDocker.outputs.tags }}
-          image_tag: ${{ steps.updatePackageJson.outputs.image_tag }}
+          image_tag: ${{ inputs.docker_image_name }}
```

Comment:
just curious, what is this change for?

Reply:
The updatePackageJson doesn't exist

Reply:
Interesting ...how the pipeline passed before 🤔

Reply:
If a variable is not found, it is just empty